server/testdriver/junit5/src/main/java/org/infinispan/server/test/junit5/InfinispanServerExtensionBuilder.java

```diff
@@ -34,6 +68,33 @@ public static InfinispanServerExtension server(String configurationFile) {
             .build();
    }
 
+   /**
+    * Starts a builder with the default configuration.
+    * Using the {@link org.infinispan.server.test.core.AbstractInfinispanServerDriver#DEFAULT_CLUSTERED_INFINISPAN_CONFIG_FILE_NAME}
+    *
+    * @return a new {@link InfinispanServerExtensionBuilder}
+    */
+   public static InfinispanServerExtensionBuilder config() {
+      return new InfinispanServerExtensionBuilder(DEFAULT_CLUSTERED_INFINISPAN_CONFIG_FILE_NAME, true);
+   }
+
+   /**
+    * Starts a builder with a custom configuration and flag for default handling.
+    *
+    * @param configurationFile config file path or a default config file name
+    * @param defaultFile whether it's the default config present in the server distribution
+    * @return a new {@link InfinispanServerExtensionBuilder}
+    */
+   public static InfinispanServerExtensionBuilder config(String configurationFile, boolean defaultFile) {
```

Comment:
Is this method really necessary? If a user is specifying a configurationFile can it really default?

Reply:
Yes, since there are different default files. For example, xsite or resp file